docs/earthfile/earthfile.md

```diff
@@ -1422,14 +1422,23 @@ The sharing mode for the cache mount, from one of the following:
 
 ##### `--chmod <octal-format>`
 
-The permission of the mounted folder, in octal format (the same format the chmod unix command line expects). 
+The permission of the mounted folder, in octal format (the same format the chmod unix command line expects).
 Default `--chmod 0644`
 
 
 ##### `--id <cache-id>`
 
 The cache ID for a global cache volume to be used across other targets or Earthfiles.
 
+##### `--persist`
+
+Make a copy of the cache available to any children that inherit from this target, by copying the contents of the cache to the child image.
+
+{% hint style='warning' %}
+Caches were persisted by default in version 0.7, which led to bloated images being pushed to registries. Version 0.8 changed the default behavior
+to prevent copying the contents to children targets unless explicitly enabled by the newly added `--persist` flag.
+{% endhint %}
```

Comment:
Can you remove this paragraph from above?

```
At the end of the target, the contents of the cache mountpoint are persisted as an additional layer in the image. This means that the contents are available to subsequent targets in the same build using `FROM`, or to any saved images `SAVE IMAGE`.
```

Reply:
I removed the warning entirely -- not sure if that's what you meant?

Reply:
No... grep for the words I quoted. Remove those :-)

Reply:
Thanks for pointing that out -- it was initially lost on me :laughing: